load-code/pe/parse-pe-exports.yml

```diff
@@ -5,7 +5,7 @@ rule:
     author: "@Ana06"
     scope: function
     mbc:
```

Comment:
This rule was deleted in the mandiant repo: https://github.com/mandiant/capa-rules/pull/487 --> should be deleted here too.